src/web/command_control/client/components/CommandControl.tsx

```diff
@@ -1651,6 +1652,9 @@ export default class CommandControl extends React.Component {
 		} else {
 			run = runs[botsAssignedToRuns[botId]]
 		}
+
```

Comment:
Since we are updating this function can we update the comment to use the correct /**/